src/amber/controller/base.cr

```diff
@@ -15,13 +15,15 @@ module Amber::Controller
     protected getter params : Amber::Validators::Params
     protected getter flash : Amber::Router::Flash::Params
     protected getter session : Amber::Router::Session::Params
+    protected getter cookies : Amber::Router::Cookies::Store?
 
     def initialize(@context : HTTP::Server::Context)
       @request = context.request
       @response = context.response
       @raw_params = context.params
       @flash = context.flash
       @session = context.session
+      @cookies = context.cookies
```

Comment:
Rails has a cookies method in the controller to easily read and write cookies. The entire purpose of writing this was to make it available to the controllers. 

http://api.rubyonrails.org/v5.1/classes/ActionDispatch/Cookies.html
http://api.rubyonrails.org/classes/ActionController/Base.html

I'm gonna move this to a method instead of the initializing it here.

```cr
def cookies
  context.cookies
end
```